flow/peerdbenv/dynamicconf.go

```diff
@@ -398,6 +428,23 @@ func PeerDBNullable(ctx context.Context, env map[string]string) (bool, error) {
 	return dynamicConfBool(ctx, env, "PEERDB_NULLABLE")
 }
 
+func PeerDBBinaryFormat(ctx context.Context, env map[string]string) (BinaryFormat, error) {
+	format, err := dynLookup(ctx, env, "PEERDB_BINARY_FORMAT")
+	if err != nil {
+		return 0, err
+	}
+	switch format {
+	case "raw":
```

Comment:
nit: make it case-insensitive since UI wouldn't really have any validation for this